css/src/style.scss

```diff
@@ -3,72 +3,44 @@
 	*/
 
 #app-navigation {
-	&:not(.vue) {
-		> ul > li {
-			> a {
-				background-position: inherit;
-			}
-			> .app-navigation-entry-edit {
-				padding-left: 5px !important;
-			}
-		}
-		.app-navigation-entry-menu {
-			ul {
-				align-items: unset;
-				li {
-					width: auto !important;
-				}
-			}
-		}
-	}
 	> ul {
 		> li {
-			background-image: unset;
-
-			> a {
+			&.collection .app-navigation-entry-icon {
```

Comment:
theoritecally you shouldn't need any custom css for your appnavigation :thinking:

Reply:
That's true, but I wanted to use the sprites file for the icons.

Reply:
Ah right, then I'm sure there is a lot of scss that can still go away ;)

Reply:
Definitly, already 100 lines SCSS less 🙂